usaspending_api/recipient/v2/views/list_recipients.py

```diff
@@ -86,10 +133,22 @@ def post(self, request):
         ]
         models.extend(copy.deepcopy(PAGINATION))  # page, limit, sort, order
 
+        # Override pagination default limit of 100
+        for model in models:
+            if model["name"] == "limit":
+                model["max"] = 1000
```

Comment:
Shouldn't the API contract be updated to include this?

Reply:
Also was there any experimentation to see if a page size higher than 1000 would be beneficial?

Reply:
I couldn't find a correct way in the API doc specification to add a max, so I just added it to the body. I originally tried something like this:
```
+ limit (optional, number)
    The number of results to include per page.
        + Default: 5
        + Max: 1000
```
But got contract errors

Reply:
In other API Contracts the maximum value is mentioned as a comment on the field. A field comment is text which is aligned with the attributes (in this example, the `+` character) but does not have any leading symbol in the text